README.md

```diff
@@ -1,11 +1,69 @@
-# storage
+# go-storage
 
 [![Build Status](https://github.com/beyondstorage/go-storage/workflows/Unit%20Test/badge.svg?branch=master)](https://github.com/beyondstorage/go-storage/actions?query=workflow%3A%22Unit+Test%22)
 [![Go dev](https://pkg.go.dev/badge/github.com/beyondstorage/go-storage/v4)](https://pkg.go.dev/github.com/beyondstorage/go-storage/v4)
 [![License](https://img.shields.io/badge/license-apache%20v2-blue.svg)](https://github.com/beyondstorage/go-storage/blob/master/LICENSE)
 [![go storage dev](https://img.shields.io/matrix/go-storage:aos.dev.svg?server_fqdn=chat.aos.dev&label=%23go-storage%3Aaos.dev&logo=matrix)](https://matrix.to/#/#go-storage:aos.dev) <!-- Need update after matrix updated -->
 
-A storage abstraction beyond the existing storage services.
+Storage abstraction that focus on neutral cross-cloud data operation.
```

Comment:
What about "vendor-neutral storage abstraction that focuses on cross-cloud data operation."?

Btw, is "cross-cloud" accurate?

Reply:
Let me think about it.

Reply:
I simplify the description to

A **vendor neutral** storage library for Golang.

Any ideas?

Reply:
But I think the hyphen is needed (vendor-neutral)?